tests/integration/page/page_enhance_within.js

```diff
@@ -0,0 +1,21 @@
+asyncTest( "Option enhanceWithin", function() {
+	$.testHelper.pageSequence([
+		function() {
+			$( "#open-enhance-within" ).click();
+		},
+		function() {
+			deepEqual( $( "#enhance-within" ).find( ":mobile-toolbar" ).length, 1,
```

Comment:
@gabrielschulhof add an id to the toolbar then check `$( "#toolbarID" ).toolbar( "instance" )` it will return the instance if it is one and false if not.